framework/src/ics/InitialConditionBase.C

```diff
@@ -8,9 +8,11 @@
 //* https://www.gnu.org/licenses/lgpl-2.1.html
 
 #include "InitialConditionBase.h"
+#include "MooseEnum.h"
 #include "SystemBase.h"
 #include "MooseVariableFE.h"
 #include "UserObject.h"
+#include "libmesh/libmesh_common.h"
```

Comment:
why this?

Reply:
I'll remove this. It must have added it automatically.